cfgov/v1/models/demo.py

```diff
@@ -28,13 +28,10 @@ class DemoPage(CFGOVPage):
 
     organisms = StreamField([
         ('well', organisms.Well()),
-        ('email_signup', organisms.EmailSignUp()),
```

Comment:
email sign up can be removeed but can you add back in the other 2 orgs full width & post preview